src/common/serviceUtils.js

```diff
@@ -0,0 +1,59 @@
+import pick from 'lodash.pick';
```

Comment:
This file is from frontend-app-account and should ultimately pulled out into the frontend-common repo